AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_HDRColorGradingAdded.py

```diff
@@ -146,42 +304,427 @@ def AtomEditorComponents_HDRColorGrading_AddedToEntity():
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Luminance Preservation')) == 1.0)
+
+        # 20. Split Toning|Weight (float range 0.0 to 1.0, default 0.0)
+        # Set Split Toning|Weight to it's maximum value to enable the parameter.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Split Toning Weight'), 1.0)
+        Report.result(Tests.split_toning_weight_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Split Toning Weight')) == 1.0)
+
+        # 21. Balance (float range -1.0 to 1.0)
+        # Set Balance to its minimum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Balance'), -1.0)
+        Report.result(Tests.balance_min_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Balance')) == -1.0)
+
+        # Set Balance to its maximum value.
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Balance'), 1.0)
+        Report.result(Tests.balance_max_value,
+                      hdr_color_grading_component.get_component_property_value(
+                          AtomComponentProperties.hdr_color_grading('Balance')) == 1.0)
+
+        # 22. Split Toning|Shadows Color (Vector3 range 0.0 to 1.0)
+        # Adjust Split Toning Shadows Color.
+        st_shadows_color = math.Vector3(0.25, 0.0, 1.0)
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Split Toning Shadows Color'), st_shadows_color)
+        st_shadows_color_value = hdr_color_grading_component.get_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Split Toning Shadows Color'))
+        Report.result(Tests.edit_split_toning_shadow_color, st_shadows_color_value.IsClose(st_shadows_color))
+
+        # 23. Split Toning|Highlights Color (Vector3 range 0.0 to 1.0)
+        # Adjust Split Toning Highlights Color.
+        st_highlights_color = math.Vector3(0.0, 1.0, 0.25)
+        hdr_color_grading_component.set_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Split Toning Highlights Color'), st_highlights_color)
+        st_highlights_color_value = hdr_color_grading_component.get_component_property_value(
+            AtomComponentProperties.hdr_color_grading('Split Toning Highlights Color'))
+        Report.result(Tests.edit_split_toning_highlights_color, st_highlights_color_value.IsClose(st_highlights_color))
+
+        # 24. Update the SMH Weight parameter (float range 0.0 to 1.0)
```

Comment:
what is SMH?

Reply:
SMH stands for "Shadows Midtones Highlights" - an abbreviation used in Editor in tooltips as well - so there is context for the use of this abbreviation.

However, in this context I should have listed that outright before referring to it as SMH (I did so in later comments, just forgot to do it here) - so I've updated this instance of it.